models/index.js

```diff
@@ -1,5 +1,13 @@
 var mongoose = require("mongoose");
+require('dotenv').config();
 
 // connect mongoose to your Mongo DB here
-
+mongoose.connection.openUri(process.env.DB_CONN);
 // add exports here
+module.exports.Book = require("./book.js");
+
+//get default connection
+var db = mongoose.connection;
```

Comment:
You don't have to set this here because you're not ever using the `db` variable yet.